src/server/database/migrations/08_initialLife.tsx

```diff
@@ -0,0 +1,9 @@
+import { DatabaseContext } from '../instance';
+
+export default {
+    identifier: '08_initialLife',
```

Comment:
Your function and identifier name doesn't seem to be related.
This calls initial life but it is creating an index